### PR TITLE
MBS-9943: Make Autocomplete.js translatable

### PR DESCRIPTION
**Pull request by reosarevok**
https://tickets.metabrainz.org/browse/MBS-9943

Change-Id: If3f1fa6982f8c809f49211c0030d840e130c5559

### CODE REVIEW

File: root/utility/releaseGroupType.js [hunk 1]
```diff
@@ -7,7 +7,7 @@
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
-import {l_attributes} from '../static/scripts/common/i18n/attributes';
+import {lp_attributes} from '../static/scripts/common/i18n/attributes';
```
Comment (reosarevok) [inline root/utility/releaseGroupType.js]: This file ended up not being used in the implementation, but this was broken anyway, so. I guess it could be split into a different commit if it matters :)

Comment (mwiencek) [reply root/utility/releaseGroupType.js]: If you don't mind. :P It's just a bit confusing when you're looking at history later or using git bisect to find out what change introduced a bug.

File: root/static/scripts/common/MB/Control/Autocomplete.js [hunk 2]
```diff
@@ -831,16 +838,18 @@ MB.Control.autocomplete_formatters = {
...
         var area = item.area;
         if (item.typeName || area) {
             a.append('<br /><span class="autocomplete-comment">' +
-                     (item.typeName ? _.escape(item.typeName) : '') +
+                     (item.typeName ? _.escape(lp_attributes(item.typeName, 'place_type')) : '') +
                      (item.typeName && item.area ? ', ' : '') +
-                     (area ? _.escape(area.name + ', ' + renderContainingAreas(area)) : '') +
+                     (area ? _.escape(area.name +
+                       (renderContainingAreas(area) ? ', ' + renderContainingAreas(area) : ''))
+                       : '') +
```
Comment (mwiencek) [inline root/static/scripts/common/MB/Control/Autocomplete.js]: Hmm, `commaOnlyList` should probably be used here too - I guess we need a create an array and push things onto it conditionally above.

File: root/static/scripts/common/MB/Control/Autocomplete.js [hunk 3]
```diff
@@ -831,16 +838,18 @@ MB.Control.autocomplete_formatters = {
...
 
         var area = item.area;
         if (item.typeName || area) {
             a.append('<br /><span class="autocomplete-comment">' +
-                     (item.typeName ? _.escape(item.typeName) : '') +
+                     (item.typeName ? _.escape(lp_attributes(item.typeName, 'place_type')) : '') +
                      (item.typeName && item.area ? ', ' : '') +
-                     (area ? _.escape(area.name + ', ' + renderContainingAreas(area)) : '') +
+                     (area ? _.escape(area.name +
+                       (renderContainingAreas(area) ? ', ' + renderContainingAreas(area) : ''))
```
Comment (mwiencek) [inline root/static/scripts/common/MB/Control/Autocomplete.js]: `area.containment && area.containment.length` would be less expensive than calling the function twice

File: root/static/scripts/common/MB/Control/Autocomplete.js (hunk 2 above)
Comment (mwiencek) [reply root/static/scripts/common/MB/Control/Autocomplete.js]: Not just for containing areas, but all the `? ', '` bits in this section.

File: root/static/scripts/common/MB/Control/Autocomplete.js [hunk 4]
```diff
@@ -13,6 +13,7 @@ const i18n = require('../../i18n');
 const commaOnlyList = require('../../i18n/commaOnlyList');
 import {l_languages} from '../../i18n/languages';
 import {lp_attributes} from '../../i18n/attributes';
+import releaseGroupType from '../../../../../utility/releaseGroupType';
```
Comment (mwiencek) [inline root/static/scripts/common/MB/Control/Autocomplete.js]: Unused import

File: root/static/scripts/common/MB/Control/Autocomplete.js [hunk 5]
```diff
@@ -22,6 +23,7 @@ const formatTrackLength = require('../../utility/formatTrackLength');
 const isBlank = require('../../utility/isBlank');
 import primaryAreaCode from '../../utility/primaryAreaCode';
 const {localStorage} = require('../../utility/storage');
+import bracketed from '../..//utility/bracketed';
```
Comment (mwiencek) [inline root/static/scripts/common/MB/Control/Autocomplete.js]: Double `//` before utility :)

File: root/static/scripts/common/MB/Control/Autocomplete.js [hunk 6]
```diff
@@ -831,17 +838,24 @@ MB.Control.autocomplete_formatters = {
...
         var area = item.area;
         if (item.typeName || area) {
+            var items =[];
+            if (item.typeName) {
+                items.push(lp_attributes(item.typeName, 'place_type'));
+            }
+            if (area) {
+                items.push(area.name);
+            }
+            if (area.containment && area.containment.length) {
```
[the remark this answers is not in the document]
Comment (mwiencek) [reply root/static/scripts/common/MB/Control/Autocomplete.js]: Nvm, pushed a fix for this plus the other small issues.
